### ISSUE TITLE
Sort methods by scope

### ISSUE THREAD

**Issue by loic-lopez** [slevomat/coding-standard#624]
Hello, thanks for creating this package it's awesome!

There is a rule to sort methods of a class by their scope for example?

```php
class Example {
   // public scope
   public $foo;
   public $foo2;

   // protected scope
   protected $bar;
   protected $bar2;
}

```

Thanks for reply 👍

**Comment by andreasschroth** [comment]
I like this idea, we also always sort methods and properties by visibility.

In your example it's properties though, not methods.

**Comment by loic-lopez** [comment]
Thanks for reply, and yes my bad i just not insert public/protected/privatemethods in my example but the main idea of sorting methods/properties by their visibilty is great i think.

**Comment by loic-lopez** [comment]
This is not possible actualy with the latest version of coding-standard?

And if not could this issue be transformed to a feature request?

**Comment by janedbal** [comment]
It is implemented here: https://github.com/cdn77/coding-standard/blob/v1.1.1/src/Cdn77/Sniffs/Classes/ClassStructureSniff.php

**Comment by loic-lopez** [comment]
@janedbal thanks for reply but how to include this sniff in ruleset.xml?
